.changeset-staged/eight-shoes-look.md

```diff
@@ -0,0 +1,17 @@
+---
+"@logto/console": patch
+"@logto/phrases": patch
+"@logto/phrases-ui": patch
+"@logto/schemas": patch
+"@logto/ui": patch
```

Comment:
should be minor since it's a new feature?

Reply:
updated